### PR TITLE
change react sample template to support web grids

### PR DESCRIPTION
**Pull request by dkamburov**

### CODE REVIEW

File: code-gen-library/WebGridGroupByRowTemplate/React.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//begin imports
+import { IgcGroupByRowTemplateContext } from 'igniteui-react-grids';
+//end imports
+
+export class WebGridGroupByRowTemplate {
+    //begin template
+    //begin content
+    public webGridGroupByRowTemplate = (ctx: IgcGroupByRowTemplateContext) => {
```
Comment (MayaKirova) [inline code-gen-library/WebGridGroupByRowTemplate/React.tsx]: Shouldn't IgcGroupByRowTemplateContext have an Igr prefix?

File: code-gen-library/WebGridGroupByRowTemplate/React.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        const values = groupRow.records;
+
+        const startDate = new Date('1/1/2017');
+        const endDate = new Date('12/31/2017');
+        var calc2017 = values.filter((x) => new Date(x.OrderDate) >= startDate && new Date(x.OrderDate) <= endDate).length;
+
+        return <><div>
+            <span style="color:#09f;">${groupRow.expression.fieldName} :</span>
+            <span>${groupRow.value}</span>
+            <igc-badge>${groupRow.records.length}</igc-badge>
```
Comment (MayaKirova) [inline code-gen-library/WebGridGroupByRowTemplate/React.tsx]: This should probably be `IgrBadge` instead.

**Review by MayaKirova — CHANGES_REQUESTED** [review]

**Review by MayaKirova — APPROVED** [review]

File: editor-templates/React/main-template/src/index.tsx [hunk 3]
```diff
@@ -14,9 +14,15 @@ import './index.css';
...
+//insert templateImports
+//end templateImports
+
+//ifdef webgrids
+import 'igniteui-react-grids/grids/combined';
+//endifdef webgrids
 
 //ifdef editor
-import 'igniteui-webcomponents/themes/light/bootstrap.css';
+import 'igniteui-react-grids/grids/themes/light/bootstrap.css';
```
Comment (HUSSAR-mtrela) [inline editor-templates/React/main-template/src/index.tsx]: @MayaKirova @dkamburov wouldn't this change force chart samples that use editor component to depend on the grids package?

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: editor-templates/React/main-template/src/index.tsx (hunk 3 above)
Comment (dkamburov) [reply editor-templates/React/main-template/src/index.tsx]: I guess so, but wasn't it like up until now?

Comment (dkamburov) [reply editor-templates/React/main-template/src/index.tsx]: Oh or we need those styles for the editor components

**Comment by ChronosSF** [comment]
@HUSSAR-mtrela , please review again so we can merge this. Thank you!

**Comment by MayaKirova** [comment]
@HUSSAR-mtrela @ChronosSF I've run a pipeline from this branch and all imports in the generated grid samples look good: https://infragistics.visualstudio.com/NetAdvantage/_build/results?buildId=111118&view=artifacts&pathAsName=false&type=publishedArtifacts

Should we also bump up the package versions to the beta 18.2.2-beta.0 since that's the first release package that has the grid or should we wait for the official release?

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**